package.json

```diff
@@ -159,8 +159,8 @@
     "@microsoft/api-extractor": "^7.52.2",
     "@peculiar/webcrypto": "^1.5.0",
     "@pmmmwh/react-refresh-webpack-plugin": "0.5.10",
-    "@rspack/cli": "^1.3.0",
-    "@rspack/core": "^1.3.0",
+    "@rspack/cli": "^1.3.6",
```

Comment:
Changes outside msw addon are already included in https://github.com/metabase/metabase/pull/57242. If that PR is merged before this one I'll rebase/merge and this change should disappear.